sdk/azcore/internal/shared/constants.go

```diff
@@ -35,5 +35,5 @@ const (
 	Module = "azcore"
 
 	// Version is the semantic version (see http://semver.org) of this module.
-	Version = "v0.21.1"
+	Version = "v0.22.0"
```

Comment:
We should be able to keep this at v0.21.1 as there were no breaking changes introduced correct?